packages/dashboard/src/components/Wizard/Wizard.utils.ts

```diff
@@ -4,10 +4,13 @@ import { SubwayNavNodeState, ISubwayNavNodeProps } from '../SubwayNav/index';
 // Get current step
 export function getCurrentStep(steps: IWizardStepProps[]): IWizardStepProps {
   let currentStep;
+  let indexToShow = -1;
 
   steps.some((wizStep: IWizardStepProps) => {
+    indexToShow++;

```

Comment:
This isn't needed since the index is an argument to our callback. Simply add an index argument here and return the index when we return true. If it's a nested item return the index of the top level plus the nested some().

Reply:
If there are few steps with sub steps, we need to consider all those indices to determine the index to show.  Hence I need to keep a count of it here.

Reply:
you would just do the same thing in that case and simply add the index from the some() of the substeps to the index from the some() of the main steps. I do this in another place. Simplifies the code.